docs/about/getting-started.md

```diff
@@ -5,37 +5,39 @@ title: Getting Started
-| --target-http-port                 | int     | 8080             | Http port for warm up requests                                                                                  |
-| --target-insecure                  | bool    | false            | Whether to skip TLS validation                                                                                  |
-| --target-readiness-path            | string  | /ready           | The path used for target readiness probe                                                                        |
-| --target-readiness-timeout-seconds | int     | -1               | Timeout for target readiness probe                                                                              |
-| --timeout-seconds                  | int     | 60               | Time after which warm up will stop making requests                                                              |
+| Flag                               | Type    | Default value    | Description                                                                                                              |
+|:-----------------------------------|:--------|:-----------------|:-------------------------------------------------------------------------------------------------------------------------|
+| --concurrency                      | int     | 2                | Number of concurrent requests for warm up                                                                                |
+| --exit-after-warmup                | bool    | false            | If warm up process should exit after completion |
+| --grpc-headers                     | strings | N/A              | gRPC headers to be sent with warm up requests.                                                                           |
+| --grpc-requests                    | strings | N/A              | gRPC request to be sent. Request is in '\<service\>\<method\>\[:message\]' format. E.g. health/ping:{"key": "value"}    |
+| --help, -h                         |         | N/A              | Help for warmup-sidecar                                                                                                  |
+| --http-headers                     | strings | N/A              | Http headers to be sent with warm up requests.                                                                           |
+| --http-requests                    | strings | N/A              | Http request to be sent. Request is in '\<http-method\>:\<path\>\[:body\]' format. E.g. post:/ping:{"key": "value"}      |
+| --probe-file-enabled               | bool    | true             | If set to true writes files to be used as readiness/liveness probes                                                       |
+| --probe-file-liveness-path         | string  | alive            | File to be used for liveness probe                                                                                       |
```

Comment:
It doesn't seem necessary to allow users to specify the names of these files. We can just document their values (ready and alive).

Reply:
It's true that it seems unnecessary. 
However, whenever it comes to files written in disk from a third-party library (like Mittens) I think users should have the option to decide on the name/filepath. This also gives them the option to write on a particular directory which would be handy from a permissions perspective.

Reply:
That's true. I was thinking of the use case where this is used as a docker sidecar (and the file is written inside the container).